apps/ledger-live-mobile/src/screens/SendFunds/07-SendBroadcastError.tsx

```diff
@@ -0,0 +1,132 @@
+  const currency = account ? getAccountCurrency(account) : null;
+
+  const error = route.params?.error;
+  const helperUrl = error?.url ?? urls.faq;
+  const gotoSupport = useCallback(() => Linking.openURL(helperUrl), [helperUrl]);
+  const exportLogs = useExportLogs();
+
+  const retry = useCallback(() => {
+    track("SendErrorRetry");
+    navigation.goBack();
+  }, [navigation]);
+
+  if (!error) return null;
+
+  return (
+    <SafeAreaView
+      style={{
+        flex: 1,
+        display: "flex",
+        flexDirection: "column",
+        marginBottom: insets.bottom,
+      }}
+    >
+      <ScrollView style={{ flex: 1, marginBottom: 16 }}>
+        <TrackScreen category="SendFunds" name="ValidationError" currencyName={currency?.name} />
+        <GenericErrorView error={error} hasExportLogButton={false} />
+
+        <Collapsible px={16} mt={24} mb={16} title={t("errors.TransactionBroadcastError.needHelp")}>
+          <Flex px={16}>
+            <InformativeBanner
+              title={t("errors.TransactionBroadcastError.helpCenter.title")}
+              description={t("errors.TransactionBroadcastError.helpCenter.desc")}
+            >
+              <InformativeBannerButton Icon={<Support />} onPress={gotoSupport}>
+                <Text fontWeight="semiBold">
+                  {t("errors.TransactionBroadcastError.helpCenter.cta")}
+                </Text>
+              </InformativeBannerButton>
+            </InformativeBanner>
+
+            <InformativeBanner
+              title={t("errors.TransactionBroadcastError.technical.title")}
+              description={error.message}
+            >
+              <Flex flexDirection="row" columnGap={8}>
+                <InformativeBannerButton Icon={<Download />} onPress={exportLogs}>
+                  <Text fontWeight="semiBold">
+                    {t("errors.TransactionBroadcastError.technical.cta")}
+                  </Text>
+                </InformativeBannerButton>
+                <InformativeBannerButton as={CopyButton} text={error.message} />
+              </Flex>
+            </InformativeBanner>
+          </Flex>
+        </Collapsible>
+      </ScrollView>
+
+      <Flex px={16}>
+        <Button type="shade" onPress={retry}>
+          {t("send.validation.button.retry")}
+        </Button>
+      </Flex>
+    </SafeAreaView>
+  );
+}
+
+type InformativeBannerProps = {
+  title: string;
+  description: string;
+  children: React.ReactNode;
+};
+
+function InformativeBanner({ title, description, children }: InformativeBannerProps) {
+  return (
+    <Card
+      p={6}
+      mb={6}
+      borderRadius={8}
+      backgroundColor="opacityDefault.c05"
+      alignItems="flex-start"
+      rowGap={16}
+    >
+      <Text numberOfLines={3} variant="paragraphLineHeight">
+        <Text>{title} :</Text> <Text color="neutral.c70">{description}</Text>
+      </Text>
+      {children}
+    </Card>
+  );
+}
+
+const InformativeBannerButton = styled(Button).attrs({
```

Comment:
Not sure since it's from the lib but can't we add an active css so the button gets a little white ? (it's done on some buttons of llm but not all of them so it's just an idea not sure that's useful)

Reply:
I realized that the active style was disabled on the `Button` component from the DS (it's not the case for quick action buttons and promisables). Here's the [commit](https://github.com/LedgerHQ/ledger-live/commit/bd4a9922f70d07d12c242d18b047cbf399592be0#diff-a09fe52ca3fe41830ba31d3682971aad9808f8f8dd4099fb94a541d0036f0aef) but I don't have the PR it was part of (this is why I sometime prefer squashing: it removes data in the git repo but makes the GH data easier to find). So I kept the change but made it overidable to hopefully not break anything 🤞.

@cgrellard-ledger do you remember the context for the change I linked ?